util/MsSql/backup-db.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/sh
 
-# Delete backup files older than 30 days
-find /etc/bitwarden/mssql/backups/ -type f -name '*.BAK' -mindepth 1 -mtime +30 -delete
+# Backup timestamp
+export now=${1:-$(date +%Y%m%d_%H%M%S)}
 
 # Do a new backup
 /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${SA_PASSWORD} -i /backup-db.sql
+
+# Delete backup files older than 30 days
+grep -B1 "BACKUP DATABASE successfully" /var/opt/mssql/log/errorlog | grep -q _$now.BAK &&
+find /etc/bitwarden/mssql/backups/ -mindepth 1 -type f -name '*.BAK' -mtime +32 -delete
```

Comment:
Why +32 now?

Reply:
To be sure to have a whole **full month** of backups.
Let's assume we are the 31st of May, you said you keep one month of backups, and someone (a manager, a coworker...) asks you to restore the database as it was on the 1st day of the month...
With 32 we're safe :)